hydrator-common/src/main/java/co/cask/hydrator/common/test/MockRealtimeContext.java

```diff
@@ -79,6 +81,12 @@ public PluginProperties getPluginProperties(String pluginId) {
   }
 
   @Override
+  public <T> T newPluginInstance(String s,
+                                 MacroEvaluator macroEvaluator) throws InstantiationException, InvalidMacroException {
+    return null;
```

Comment:
then, I'd have to split the exceptions to the next line. I just thought the split here looked cleaner.